config/300-configmappropagation.yaml

```diff
@@ -0,0 +1,62 @@
+# Copyright 2019 The Knative Authors
```

Comment:
Now that #2428 has merged, this file should be moved to `config/core/resources/configmappropagation.yaml`, with a symlink `config/300-configmappropagation.yaml` pointing there.

Reply:
done.